components/crypto_exchange/browser/buildflags/buildflags.gni

```diff
@@ -1,7 +1,8 @@
 import("//brave/build/config.gni")
 import("//brave/components/binance/browser/buildflags/buildflags.gni")
+import("//brave/components/brave_together/buildflags/buildflags.gni")
 import("//brave/components/gemini/browser/buildflags/buildflags.gni")
 
 declare_args() {
-  crypto_exchanges_enabled = binance_enabled || gemini_enabled
+  crypto_exchanges_enabled = binance_enabled || gemini_enabled || brave_together_enabled
```

Comment:
I think it would make sense to rename `crypto_exchanges_enabled` given this

Reply:
maybe to `common_widget_utils_enabled`?

Reply:
i went for `ntp_widget_utils` for the component and renamed everything